src/test/scala/chiselTests/BlackBox.scala

```diff
@@ -150,23 +150,28 @@ class BlackBoxWithParamsTester extends BasicTester {
 class BlackBoxSpec extends ChiselFlatSpec {
   "A BlackBoxed inverter" should "work" in {
     assertTesterPasses({ new BlackBoxTester },
-        Seq("/chisel3/BlackBoxTest.v"))
+        Seq("/chisel3/BlackBoxTest.v"),
+      annotations = TesterDriver.verilatorOnly)
```

Comment:
style nit: any reason this line isn't aligned with the above?

Reply:
Also, in some cases, you use `annotations = ...`, and in others, you don't have the explicit keyword. Is this by choice (and what's the underlying rationale) or chance?

Reply:
I just followed the previous alignment here. But it is pretty messed up so I have reformatted.
Removed using named parameter